common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java

```diff
@@ -165,17 +165,17 @@ void transmissionRiskLevelDoesNotThrowForValid(int validRiskLevel) {
   }
 
   @ParameterizedTest
-  @ValueSource(ints = {-3, 143, 145})
+  @ValueSource(ints = {-3, 145})
   void rollingPeriodMustBeEpectedValue(int invalidRollingPeriod) {
     assertThat(catchThrowable(() -> keyWithRollingPeriod(invalidRollingPeriod)))
         .isInstanceOf(InvalidDiagnosisKeyException.class)
-        .hasMessage("[Rolling period must be " + DiagnosisKey.EXPECTED_ROLLING_PERIOD
+        .hasMessage("[Rolling period must be between "+ DiagnosisKey.MIN_ROLLING_PERIOD + " and " + DiagnosisKey.MAX_ROLLING_PERIOD
             + ". Invalid Value: " + invalidRollingPeriod + "]");
   }
 
   @Test
   void rollingPeriodDoesNotThrowForValid() {
-    assertThatCode(() -> keyWithRollingPeriod(DiagnosisKey.EXPECTED_ROLLING_PERIOD)).doesNotThrowAnyException();
+    assertThatCode(() -> keyWithRollingPeriod(DiagnosisKey.MAX_ROLLING_PERIOD)).doesNotThrowAnyException();
```

Comment:
should there be more test? Like for example for `DiagnosisKey.MIN_ROLLING_PERIOD` and a value between `MIN` and `MAX`?